src/components/Headerstrip/Headerstrip.js

```diff
@@ -47,25 +69,43 @@ class Headerstrip extends Component {
   }
 
   onAccept = event => {
-    event.preventDefault()
+    const { onAccept } = this.props
     if (typeof this.props.onAccept === 'function') {
```

Comment:
```suggestion
    if (onAccept) {
```

Ya está asignada la variable en el scope y no es necesario comprobar el tipo porque ya está validado con prop types, que solo puede ser función (el dev igual podría pasar cualquier cosa e ignorar el warning, pero eso ya es mal uso de la herramienta).

Reply:
Si, concuerdo con que debería funcionar sólo con los prop types, pero como dices no todos lo que la utilizan le dan un buen funcionamiento, y a mi juicio creo que es mejor prevenir este tipo de cosas en la medida que se pueda hacer, además este es un proyecto público creo que con mayor razón deberíamos prevenir
Con lo otro si! voy a arreglar porque ya esta asignada al scope.